File: Fibonacci/Fibonacci.cs
using System;
using System.IO;

class Hello
{
    static void Main()
    {
        int i= 0, j= 1, s= 0;
        int cont = 2;
        int quantidade = 0;
        

        Console.WriteLine("Digite a quantidade de valores Fibonacci desejaveis");
        quantidade = Convert.ToInt32(Console.ReadLine());

        Console.WriteLine($"{i}");
        Console.WriteLine($"{j}");

        while(cont < quantidade)
        {

            s = i + j;
            Console.WriteLine($"{s}");
            i = j;
            j = s;
            cont++;
        }
        
    }
}

File: Fibonacci/Fibonacci.java
public class HelloWorld{

	public static void main(String[] args){

		int i = 0;
		int j = 1;
		int soma = 0;
		int cont = 2;
		int quantidade = 0;

		// Simulação de uma entrada 
		quantidade = 15;

		System.out.println("Sequencia de Fibonacci");
		System.out.print(i);
		System.out.print(" --> ");
		System.out.print(j);

		while(cont < quantidade){
			soma = i + j;
			System.out.printf(" --> %d",soma);
			i = j;
			j = soma;

			cont++;

		}



	}

}

